index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TechHub-Careers</title>
+    <!-- Css  -->
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="nav-container">
+            <div id="nav-branding">
+                <h1>Tech<span>Hub</span>-Careers</h1>
+            </div>
+            <nav>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="#">Careers</a></li>
+                    <li><a href="#">Wait-List</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <section></section>
+    <footer>
+        <div class="box contact">
+            <h3>Tech<span>Hub</span>-Careers</h3>
+            <hr class="light">
+            <p>info@techhub-careers.com</p>
+            <p>Niger II</p>
+            <p>Dufuna Code-Camp20</p>
+        </div>
+        <div class="box search">
+            <hr class="light">
+                <h3>Leverage on Tech<span>Hub</span>-Careers</h3>
+                <hr class="light">
+                <input type="text" name="search" id="search" placeholder="What are you looking for?">
+                <a href="#" target="_blank" rel="Search">
+                    <button type="submit" class="btn" name="search" id="search">Search</button>
+                </a>
+                
+        </div>
+        <div class="box social-media">
+            <h3>Contact Us</h3>
+            <hr class="light">
+            <div class="icon">
+                <a href="#"><img src="./image/facebook-logo.png" alt="Facebook"></a>
+                <a href="#"><img src="./image/instagram-logo.jpg" alt="Instagram"></a>
+                <a href="#"><img src="./image/medium-logo.png" alt="Medium"></a>
+                <a href="#"><img src="./image/twiiter-logo.png" alt="Twitter"></a>
+            </div>
+        </div>
+        <div class="copywrite">
```

Comment:
don't forget to update the class name as well
```suggestion
        <div class="copyright">
```